pkg/tinyusb/contrib/tinyusb.c

```diff
@@ -27,6 +27,11 @@ static void *_tinyusb_thread_impl(void *arg)
 {
     (void)arg;
 
+    if (tinyusb_hw_init() != 0) {
+        DEBUG("tinyUSB peripherals couldn't be initialized\n");
+    }
+    DEBUG("tinyUSB peripherals initialized\n");
+
```

Comment:
In the current state, if the init failed, the output will be confusing as both lines will be printed.
Maybe use a else statement or add a return ?

Reply:
Good catch. I removed the `return` statement by mistake when moving the code from `tinyusb_setup` to `*_tinyusb_thread_impl`. With commit 264eba2c7c4104a7875b071c4fd35c0353ae09fa I replaced the `DEBUG` output by `LOG_ERROR` output in case of error and added an `assert` in case of error in hwardware initialization.